src/exodus/test/sigmadb_tests.cpp

```diff
@@ -427,4 +429,31 @@ BOOST_AUTO_TEST_CASE(use_differnet_group_size_from_database)
     );
 }
 
+BOOST_AUTO_TEST_CASE(test_notification)
```

Comment:
Seem like we don't need this test if we remove subscription.

Reply:
removed